lib/mini_twitter_client.rb

```diff
@@ -0,0 +1,43 @@
+require 'faraday'
+require 'json'
+
+class MiniTwitterClient
+  def connection
+    Faraday.new(
+      url: 'http://localhost:4000/api/tweets',
+      params: {},
+      headers: {
+        'Accept' => 'application/json',
+        'Content-Type' => 'application/json'
+      }
+    )
+  end
+
+  def fetch_tweets
+    response = connection.get
+
+    JSON.parse(response.body)['data'].map do |tweet|
+      {
+        author: tweet['author'],
+        id: tweet['id'],
+        message: tweet['message']
+      }
+    end
+  end
+
+  def create_tweet(author, message)
+    connection.post { |post| post.body = { "tweet": { "author": author, "message": message } }.to_json }
```

Comment:
`fetch_tweets` method returns parsed tweets.

I think this method could behave in similar way:

```suggestion
    response = connection.post { |post| post.body = { "tweet": { "author": author, "message": message } }.to_json }
    tweet = JSON.parse(response.body)['data']
    {
      author: tweet['author'],
      id: tweet['id'],
      message: tweet['message']
    }
```